docs/framework/wcf/feature-details/how-to-serialize-and-deserialize-json-data.md

```diff
@@ -5,15 +5,15 @@ ms.assetid: 88abc1fb-8196-4ee3-a23b-c6934144d1dd
 ---
 # How to: use DataContractJsonSerializer
```

Comment:
(Same in metadata)

```suggestion
# How to use DataContractJsonSerializer
```

<https://github.com/dotnet/docs/blob/master/styleguide/template.md#headings>

> Don't add a colon after "How to" (for example, "How to sort an array" and not "How to: Sort an array").